src/lua/lua_manager.cpp

```diff
@@ -303,6 +316,9 @@ namespace big
 			return {};
 		}
 
+		if(std::filesystem::relative(module_path.parent_path(), m_scripts_folder.get_path()).string().contains("includes"))
```

Comment:
what's the reasoning behind this?

Reply:
Some scripts are library scripts, they do nothing on their own and are intended to be used with `require`, they take up space in the script list for no reason.